start-miner.sh

```diff
@@ -15,6 +15,9 @@ if [ "$BALENA_DEVICE_TYPE" = "rockpi-4b-rk3399" ]; then
 elif [ "$BALENA_DEVICE_TYPE" = "intel-nuc" ]; then
   cp /opt/miner/docker.config.5g "/opt/miner/releases/$HELIUM_GA_RELEASE/sys.config"
   OVERRIDE_CONFIG_URL="${5G_MINER_CONFIG_URL:-https://helium-assets.nebra.com/docker.config.5g}"
+elif [ "$VARIANT" = "COMP-PISCESP100" ]; then
```

Comment:
A little weird to mix `BALENA_DEVICE_TYPEBALENA_DEVICE_TYPE` and `VARIANT`. Could we switch to VARIANT everywhere?

Reply:
@marvinmarnold we could - but then it would require a lot more versions - as we have indoor and outdoor variants for nebra rockpi and nebra raspberry pi, and we have rak, sensecap, finestra, controllino and others that can all use the same config and just the balena version.

pisces is raspberry pi, but for some strange reason they have put the ecc on i2c-0 (very odd). so it is an outlier in that sense.

so think it probably makes sense to do it this way - but not sure what you think?

Reply:
I find the miner config loading pipeline extremely confusing. I do think it would be more maintainable to enumerate variants than keep track of which miners are running on which hardware. But maybe that would be best done by refactoring all of this into a Python script. That's out of scope for this PR so I defer to you.

But one thing that is potentially blocking. Are you saying that Pisces is also rpi and so will also evaluate true for `"$BALENA_DEVICE_TYPE" = "intel-nuc"`. In that case, this line will never run and needs to be placed above.

Reply:
@marvinmarnold why would the intel nuc line block this?

I think the logic is to set as Raspberry Pi as the default and then if it's a RockPi, Intel nuc or pisces to override.

Pisces is Raspberry Pi so this will work fine 

However I do agree that it might be better to do as variants (can just use OR statements I guess for now and then later pythonise).

Or alternatively we just leave it as is for now and pythonise when moving to light, to save time and not spend cycles on code that will be deprecated soon

Reply:
Ah right. Despite my current confusion kind of proving the point, I think it's fine to leave as-is and keep things moving.

Reply:
Ok I'll merge this then and then add two new issues for the variants, and the pythonisation 👌